app/scripts/lib/setupSentry.js

```diff
@@ -30,6 +30,53 @@ export const SENTRY_BACKGROUND_STATE = {
   AccountTracker: {
     currentBlockGasLimit: true,
   },
+  PermissionLogController: {
+    // double-check if contains PID?
+    permissionHistory: true,
+    // double-check if contains PID?
+    permissionActivityLog: true,
+  },
+  AnnouncementController: {
+    announcements: true,
+  },
+  GasFeeController: {
+    gasFeeEstimates: true,
+    estimatedGasFeeTimeBounds: true,
+    gasEstimateType: true,
+  },
+  TokenListController: {
+    // double-check if contains PID?
+    tokenList: true,
```

Comment:
This one is borderline, I don't think we should include it in its entirety at least. Probably safest to exclude for now. Same for `tokensChainsCache`

Reply:
Resolved in https://github.com/MetaMask/metamask-extension/pull/20595/commits/b3330828235d61e68ab356e3e859a30d3c44af6f